content/en/security/application_security/setup/python/troubleshooting.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Troubleshooting Python App and API Protection
+further_reading:
+  - link: "/security/application_security/how-it-works/"
+    tag: "Documentation"
+    text: "How App and API Protection Works"
+  - link: "/security/default_rules/?category=cat-application-security"
+    tag: "Documentation"
+    text: "OOTB App and API Protection Rules"
+  - link: "/security/application_security/troubleshooting"
+    tag: "Documentation"
+    text: "Troubleshooting App and API Protection"
+---
+
+## Common Issues
+
+### No security signals appearing
+
+1. **Verify Agent version**
+   - Ensure you're running Datadog Agent v7.41.1 or higher
+   - Check Agent status: `datadog-agent status`
+
+2. **Check Python tracer version**
+   - Confirm you're using Python tracer v1.9.0 or higher
```

Comment:
It's too low, isn't it?
Would be better to say: v2.21 or a recent version on v3 ?